src/old/lib/utilities/API/api.ts

```diff
@@ -303,3 +303,7 @@ export const updatePlatformInformation = async (
 ): Promise<AxiosResponse<PlatformInformationType>> => {
   return instance.put(`/platform-information/`, requestBody);
 };
+
+export const getDoctorsList = async (): Promise<AxiosResponse> => {
```

Comment:
but why without subtype for `AxiosResponse<...>`? and where is it used?

Reply:
unnecessary method, have already removed it